integrations/oidc/src/index.tsx

```diff
@@ -351,9 +339,12 @@ const handleFetchEvent: FetchEventCallback<OIDCRuntimeContext> = async (request,
                         );
                     }
                 } else {
-                    return new Response('Error: Either ClientId or Client Secret is missing', {
-                        status: 400,
-                    });
+                    return new Response(
+                        'Error: Either ClientId or Client Secret or Access Token Endpoint is missing',
```

Comment:
We can easily make the message more specific with a conditional. I think it's worth it, wdyt?

Reply:
All these things are configured from the same screen, so even if make the message more specific, the user will see the other items as well (missing or not).

I don't think it's worth it tbh since when any of these is missing, the user will see all the other items as well